openassessment/data.py

```diff
@@ -1168,8 +1169,26 @@ def create_zip_with_attachments(cls, file, submission_files_data):
         csvwriter = csv.DictWriter(csv_output_buffer, cls.SUBMISSIONS_CSV_HEADER, extrasaction='ignore')
         csvwriter.writeheader()
 
+        logger.info("Beginning writing to ZIP file")
```

Comment:
Does the logger give contextual information like [this is coming from ORA attempting to create the submission archive]? I could see this message being a bit vague without it.

Reply:
Yeah, I agree that this isn't particularly useful out of context, but in the specific use case of  "hey, does this method get called? Where the heck does execution end?" it has a purpose

Reply:
This task is called infrequently enough that there's very very little chance of overlap

Reply:
I've got https://openedx.atlassian.net/browse/EDUCATOR-5886 to address post-diagnosis cleanup